01-messaging.md

```diff
@@ -494,6 +494,76 @@ every message maximally).
 Finally, the usage of periodic `ping` messages serves to promote frequent key
 rotations as specified within [BOLT #8](08-transport.md).
 
+### Peer storage
```

Comment:
I think the "Peer Storage" header should be larger that the sub-headers. Ie, main header should use "##" and subheaders "###". 

Also - should add an entry to the table of contents at the top of the doc

Reply:
Thanks for pointing this out. :)